### ISSUE TITLE
Log Emptying the Trash

### ISSUE THREAD

**Issue by sottwell** [modxcms/revolution#12454]
Emptying the Resource Tree's trash is not logged in the Manager Actions. It's a simple one line addition to the emptyrecyclebin.class.php file around line 59 -    
```
$this->modx->logManagerAction('Empty Trash','modResource','Trash');
```
![screen shot 2015-05-23 at 7 27 06 am](https://cloud.githubusercontent.com/assets/1073780/7782371/2ea35178-011d-11e5-9a61-26bc4ef9e0f8.png)

**Comment by OptimusCrime** [comment]
I'll take a look at this. @optimuscrime

**Comment by OptimusCrime** [comment]
Fixed in #eb544689d25495fbc1bf5918196fea5bfe4dd034

**Comment by OptimusCrime** [comment]
Fixed in #12456

@modxbot close
